readme.md

```diff
@@ -192,13 +192,22 @@ This also accepts an object with separate `connect`, `socket`, and `request` fie
 
 ###### retries
 
-Type: `number` `Function`<br>
-Default: `2`
+Type: `number` `Function` `Object`<br>
+Default:
+```js
+{
+	retry: 2,
+	methods: ['GET', 'PUT', 'HEAD', 'DELETE', 'OPTIONS', 'TRACE'],
+	statusCodes: [408, 413, 429, 502, 503, 504]
```

Comment:
I think we should link up the status codes (like I did in the issue), so people can quickly check what they mean. Would save them some time.